controllers/prep_handlers.go

```diff
@@ -234,6 +235,10 @@ func (r *ImageBasedUpgradeReconciler) SetupStateroot(ctx context.Context, ibu *l
 		return fmt.Errorf("failed rpm-ostree cleanup -b: %w", err)
 	}
 
+	if err := reboot.WriteIBUAutoRollbackConfigFile(ibu, r.OstreeClient); err != nil {
```

Comment:
Should we consider including all this configuration in the `SeedReconfiguration` struct? I feel the pre-pivot LCA and post-pivot LCA communications are getting a bit scattered

Reply:
On the other hand SeedReconfiguration is shared with IBI, which doesn't care about rollbacks, so you could consider it to be polluting the struct. 

wdyt?

Reply:
Yeah, I was trying to keep out of the IBI area as much as possible, hence the separate config file. In IBI, the file won't be present, so it would be skipped.